script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt

```diff
@@ -88,14 +88,12 @@ One leader commanding a large number of workers... something along those lines?"
 -- Page 526, Offset 17877.
 -- 前にも話したけど、こういう階梯」
 // IT'S WHITEBOARD TIME FUCK MY LIFE!!!! ( ͡° ͜ʖ ͡°)
-We touched on this before, but here are the Ranks."
+We touched on this before, but here are the ranks."
 }
 [sha:0359e2d3816f1a461a5e7e3dab3b85db2c6ddca3]{
 -- Page 527, Offset 17878.
 -- 「まずⅠ階梯、死者。
-// 死者 are members of the group 死者 but that's weird and confusing so for purposes of whiteboarding,
-// rank 1 dead are 'Corpses' and the group containing ranks 1-3 is the Dead.
-"First we have Rank I, Corpses.
+"First we have Rank I, the Dead.
```

Comment:
It's Rank I + II + III = 死者
And yeah, because I don't want us to get in a situation where we have to always think about where 死者 is talking about Rank I or the grouping. Now it's ambiguous in the exact same way as the original Japanese.